.github/workflows/actions.yml

```
@@ -0,0 +1,39 @@
+name: CI
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: ['2.5', '2.6', '2.7', '3.0']
+        activerecord: ['4.2', '5.0', '5.1', '5.2']
```

There are only two gemfiles, right?
```suggestion
        activerecord: ['4.2', '5.2']
```